source/module_orbital/ORB_read.h

```diff
@@ -97,7 +97,8 @@ class LCAO_Orbitals
 	int lmax_d;	//max l of descriptor orbitals
 	int nchimax_d;	//max number of descriptor orbitals per l
 
-	double rcutmax_Phi;	//caoyu add 2021-05-24
+    double rcutmax_Phi = 0.0;
```

Comment:
The usual practice is to initialize the member variables of a class in its constructor.